src/Screens/Bookmark/BookmarkIcon.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Shouldn't this file be under the shared components folder?

Reply:
The only place that calls it now is the breadcrumb.  If other so technically it could go under header.

Reply:
Sure, I'm okay with it being moved there.

Reply:
Okay I'm moving BookmarkIcon to Header folders